_posts/2018-01-13-DAY-4-solution-in-go.md

```diff
@@ -0,0 +1,116 @@
+---
+published: true
```

Comment:
Add title, author, layout and date

[Read this](https://jekyllrb.com/docs/posts/)